recipes/spirv-tools/all/conanfile.py

```diff
@@ -88,13 +44,8 @@ def configure(self):
     def layout(self):
         cmake_layout(self, src_folder="src")
 
-    def _require(self, recipe_name):
-        if recipe_name not in self._dependencies_versions:
-            raise ConanException(f"{recipe_name} is missing in {self._dependencies_filename}")
-        return f"{recipe_name}/{self._dependencies_versions[recipe_name]}"
-
     def requirements(self):
-        self.requires(self._require("spirv-headers"))
```

Comment:
@SpaceIm 
This breaks updates with spirv-headers as they use a different version number system with the latest being: "1.3.243.0"
The bot reports the following trying to update it: 
"Failed requirement 'spirv-headers/2023.2' from 'spirv-tools/2023.2'
ERROR: Unable to find 'spirv-headers/2023.2' in remotes"

I think we should revert to the previous method of mapping the spirv-tools version to the spirv-headers one

Reply:
Which bot? This PR passed v1 & v2 pipeline in c3i. We don't package 2023.2 in conan-center, only SDK versions.